pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -95,6 +100,23 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	err = c.Watch(&source.Kind{Type: &batchv1.Job{}}, &handler.EnqueueRequestForOwner{
```

Comment:
the previous one (L95-101) can be deleted now I suppose

Reply:
I think we should still watch pods, like the blocker pods

Reply:
as discussed, we'll remove it as we don't need to watch the blocker pods. all other pods got moved to jobs

Reply:
done